### DISCUSSION TITLE
FHIR to v2

### DISCUSSION THREAD

**Discussion by bkruegermoxe** [microsoft/FHIR-Converter#314] [Q&A: Handlebars Engine]
Am I correct in assuming that all of the templates would have to be created in order to support converting FHIR to v2, or is there a lot more involved than just that?

**Comment by ACMoretxj** [comment]
Template is not the only thing needed to do to convert FHIR resources to V2 messages. The engine should be able to parse the FHIR resources firstly, and then the templates are used to render V2 messages.

**Reply by mikaelweave** [reply]
Any plans or thoughts on the FHIR to V2 conversion? From my experience, this seems necessary for write, EMR integrated FHIR applications in Azure.
